app/src/main/res/values/strings.xml

```diff
@@ -1031,4 +1031,6 @@
     <string name="pref_flashlight_toggle_with_system_title">Toggle with system</string>
     <string name="pref_flashlight_toggle_with_system_summary">Displays a notification when the flashlight is turned on outside of Trail Sense</string>
     <string name="pref_flashlight_settings">pref_flashlight_settings</string>
+    <string name="pref_navigation_lock_screen_presence" translatable="false">pref_navigation_show_on_lock_screen</string>
+    <string name="pref_navigation_lock_screen_presence_title">Presence on lock screen while navigating</string>
```

Comment:
Can this be renamed to "Keep unlocked while navigating"?

Reply:
Hmm it's not locked... 
I'm not sure, but "Keep unlocked while navigating" is strictly speaken not correct (but I can change it nevertheless, if you want)

Reply:
Okay, let's keep it like this for now, and I'll change it if I find a better term